package.json

```diff
@@ -35,7 +35,7 @@
   },
   "license": "MIT",
   "dependencies": {
-    "axios": "^0.18.0",
+    "gaxios": "^1.0.2",
```

Comment:
Q: Is the idea that in a future semver major we will drop this and just use `node-fetch`? Or is `gaxios` going to be a permanent feature?

Reply:
I put this in the same camp as teeny-request.  I think we're going to end up needing this long term.  There are a few reasons:
- *Proxy support*.  This is something we hacked up in google-auth-library, but I want to have it in the underlying HTTP client.
- *Retry logic*.  There may be a `retry-fetch` out there, but it's actually kind of nice to have this just bundled in as a feature. 
- *Complex HTTP features*.  I suspect `node-fetch` will remain light on features (which is a feature).  As we need things like multipart requests, it is nice to have a single place to add them.